compiler/qsc/src/interpret/stateful/tests.rs

```diff
@@ -1236,16 +1238,11 @@ mod given_interpreter {
             ];
 
             let sources = SourceMap::new(sources, None);
-            let interpreter = Interpreter::new(
-                true,
-                sources,
-                PackageType::Lib,
```

Comment:
You'll notice that the test used to pass `PackageType::Lib` here, but the new `Debugger` always passes in `PackageType::Exe` at initialization. I think this change made sense because you can't actually debug a `Lib` in practice (it will always fail with no entrypoint error) , but I'm open to feedback if I missed anything